kolibri/core/kolibri_plugin.py

```diff
@@ -38,6 +38,7 @@ class KolibriCoreFrontEnd(KolibriFrontEndPluginBase):
     """
     entry_file = "assets/src/kolibri_core_app.js"
     external = True
+    core = True
```

Comment:
(Looks like `KolibriCore` is never used, and currently contains what looks like an example subclass implementation?)